pkg/cvo/sync_worker.go

```diff
@@ -620,6 +627,11 @@ func (w *SyncWorker) apply(ctx context.Context, payloadUpdate *payload.Update, w
 				continue
 			}
 
+			if shouldExclude(w.exclude, task.Manifest) {
```

Comment:
syncwork already has an override here https://github.com/openshift/cluster-version-operator/pull/252/files#diff-b2099007e7494e37fc4f7a4320f3622bR68

and exclude very similar to that, is it possible we can unify that? they seem to have pretty similar fingerprint??